test/Libraries/RevitIntegrationTests/AsyncRunningTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.IO;
+using System.Linq;
+using Autodesk.Revit.DB;
+using Dynamo.Core.Threading;
+using DSCoreNodesUI.Input;
+using NUnit.Framework;
+using Revit.Elements;
+using RevitServices.Persistence;
+using RevitTestServices;
+using RTF.Framework;
+using Dynamo.Nodes;
+
+namespace RevitSystemTests
+{
+    /// <summary>
+    /// Test cases in this fixture will be supposed to run in an asyncronous mode
```

Comment:
It is the scheduled tasks that are going to run asynchronously. That is the reason I am saying "run in an asyncronous mode".